IronSoftware.Drawing/IronSoftware.Drawing.Common/Color.cs

```diff
@@ -4,33 +4,42 @@
 
 namespace IronSoftware.Drawing
 {
+    // <summary>
+    /// <para>A universally compatible Color for .NET 7 and .NET 6, .NET 5, .NET Core. Windows, NanoServer, IIS,  macOS, Mobile, Xamarin, iOS, Android, Google Compute, Azure, AWS and Linux compatibility.</para>
+    /// <para>Works nicely with popular Image Color such as <see cref="System.Drawing.Color"/>, <see cref="SkiaSharp.SKColor"/>, <see cref="SixLabors.ImageSharp.Color"/>, <see cref="Microsoft.Maui.Graphics.Color"/>.</para>
+    /// <para>Implicit casting means that using this class to input and output Color from public API's gives full compatibility to all Color type fully supported by Microsoft.</para>
+    /// </summary>
     public partial class Color
     {
 
         /// <summary>
-        /// Gets the alpha component value of this System.Drawing.Color structure.
+        /// Gets the alpha component value of this IronSoftware.Drawing.Color structure.
         /// </summary>
         /// <return>The alpha component value of this IronSoftware.Drawing.Color.</return>
         public byte A { get; internal set; }
 
         /// <summary>
-        /// Gets the green component value of this System.Drawing.Color structure.
+        /// Gets the green component value of this IronSoftware.Drawing.Color structure.
         /// </summary>
         /// <return>The green component value of this IronSoftware.Drawing.Color.</return>
         public byte G { get; internal set; }
 
         /// <summary>
-        /// Gets the blue component value of this System.Drawing.Color structure.
+        /// Gets the blue component value of this IronSoftware.Drawing.Color structure.
         /// </summary>
         /// <return>The blue component value of this IronSoftware.Drawing.Color.</return>
         public byte B { get; internal set; }
 
         /// <summary>
-        /// Gets the red component value of this System.Drawing.Color structure.
+        /// Gets the red component value of this IronSoftware.Drawing.Color structure.
         /// </summary>
         /// <return>The red component value of this IronSoftware.Drawing.Color.</return>
         public byte R { get; internal set; }
 
+        /// <summary>
+        /// Construct a new CropRectangle.
```

Comment:
@mee-ironsoftware This looks like a misplaced XML comment.

Reply:
Perhaps:
`        /// Construct a new Color.` ?

Reply:
Yes sir, you're correct.